workflow/controller/controller.go

```diff
@@ -145,16 +149,20 @@ func (wfc *WorkflowController) Run(ctx context.Context, wfWorkers, podWorkers in
 
 	wfc.wfInformer = util.NewWorkflowInformer(wfc.restConfig, wfc.Config.Namespace, workflowResyncPeriod, wfc.tweakWorkflowlist)
 
+	informerFactory := wfextv.NewSharedInformerFactory(wfc.wfclientset, workflowTemplateResyncPeriod)
+	wfc.wftmplInformer = informerFactory.Argoproj().V1alpha1().WorkflowTemplates()
+
 	wfc.addWorkflowInformerHandler()
 	wfc.podInformer = wfc.newPodInformer()
 
 	go wfc.wfInformer.Run(ctx.Done())
```

Comment:
Will the old wfInformer also watch and get wf info from apiServer ?

Reply:
I think `wfInformer` is only for `Workflows`.
https://github.com/argoproj/argo/blob/master/workflow/util/util.go#L57

Reply:
Yep,   LGTM